dashboard/src/components/ChartView/ChartVersionsList.tsx

```diff
@@ -21,13 +21,17 @@ class ChartVersionsList extends React.Component<IChartVersionsListProps, IChartV
   public render() {
     const versions = this.state.showAll ? this.props.versions : this.props.versions.slice(0, 5);
     const items = versions.map(v => {
+      const chartData = v.relationships.chart.data;
       const selectedClass =
         this.props.selected.attributes.version === v.attributes.version
           ? "type-bold type-color-action"
           : "";
+      // TODO(absoludity): the namespace in the url call below should be the current namespace, not the repo namespace.
+      // For example, if a global chart is being viewed from `my-namespace`, we don't want to switch to the `kubeapps`
+      // namespace when selecting a specific chart version (as the app currently does).
```

Comment:
this is a bug, is there anything preventing you from fixing it now? (I'm worried we forget about this)

Reply:
Just that I wasn't sure how many hoops I'd need to jump through to pass the namespace down at the time, so was going to do it in a followup PR. Just checked though and namespace is available in the parent `ChartView` component, so it's trivial - done.